### ISSUE TITLE
Show full prediction time series, not just tomorrow onwards

### ISSUE THREAD

**Issue by jaryaman** [pr3d4t0r/COVIDvu#465]
Plots like this don't look too great (Italy)
![image](https://user-images.githubusercontent.com/7837158/78340064-5f6c6800-758d-11ea-8dce-10b5e6056bcc.png)

This is happening because the trend for the most recent few days don't quite fit the history. The model weights every data point equally, whereas plots like this beg us to give a higher weight to today's data. 

If we show the full prediction time series, especially on a log scale, the eye is more forgiving, and more closely reflects what's going on under the hood with our predictions.

![image](https://user-images.githubusercontent.com/7837158/78340198-93478d80-758d-11ea-9f2b-eb42c26787fd.png)

Suggestion: When toggling "show predictions", show:
- The full predicted time series (including past values)
- Default to showing on a log scale

@kinabalu @pr3d4t0r thoughts?

**Comment by kinabalu** [comment]
Bumping this back up, the full scale we can do, though the Show Predictions is only shown when you are in the Linear or Log mode, and doesn't have "a default". So we want to revert back to showing all time series data that we have for predictions? @jaryaman

**Comment by jaryaman** [comment]
@kinabalu yes please, displaying the full prediction time series I think helps...

One thing to be mindful of is whether it will still be obvious which date is today when we revert this visualisation. Some ideas:
- Plot the data last in plotly, so it's the upper-most layer (I think we do this already, just checking)
- Make the predictions sufficiently light in shade, or sufficiently high transparency/alpha
- Make the data marker size sufficiently large

My least-preferred option would be to re-introduce the "You are here" marker on the plots. But I kinda feel that's a crutch -- ideally it'd be obvious which day is today without having to put a landmark.
